home/tests.py

```diff
@@ -1,3 +1,79 @@
+from home.models import Profile, Subject, Question, Tag, Question_Tag
+from django.contrib.auth.models import User
+from django.utils import timezone
+import pytest
+
 
 def test_first():
     assert True
+
+
+@pytest.mark.django_db
+def test_add_tags_to_question():
+    # create test data
+    test_question, test_tags = create_test_data()
```

Comment:
I would leave the definition of "tags" in the body of the test function so its easier to find out what you are passing to the function you are testing.